__tests__/__mocks__/data/mockApiResponses.tsx

```diff
@@ -0,0 +1,86 @@
+export const mockFeaturedItemResponse = {
+  headers: { status: "success", code: "200", message: "ok" },
+  numResults: "1",
+  capture: {
+    uuid: "510d47d9-7c7c-a3d9-e040-e00a18064a99",
+    imageLinks: { imageLink: [Array] },
```

Comment:
I'm surprised this doesn't throw an error when it's run. Replace `Array` with mocked data.